pizza_recipe.txt

```diff
@@ -15,8 +15,10 @@ Make eight dough balls, about 16 oz each.
 
 STONE PIZZA
  ----------
-preheat oven to 550 degrees F.
+preheat oven to 425 degrees F.
 make pizza on stone.
-bake for 9 minutes.
+bake for 10-15 minutes.
```

Comment:
Yikes!  This might burn my pizza.  I like my pizza less crispy.  Can we do 8-10 minutes?